src/utils/__tests__/Xhr-test.js

```diff
@@ -301,113 +301,39 @@ describe('util/Xhr', () => {
     });
 
     describe('shouldRetryRequest', () => {
-        const createXhrInstance = shouldRetryRequest => {
+        const createXhrInstance = options => {
             xhrInstance = new Xhr({
                 token: '123',
-                shouldRetryRequest,
```

Comment:
This was not the right name for the option - it is `shouldRetry`. The first test case below was only passing because retry count was set to 4 initially.